### PR TITLE
WIP: Non-pharmaceutical intervention data

### PR DESCRIPTION
**Pull request by cc7768**
This PR incorporates the NPI data collected by Keystone to our API

The only remaining todo is to write documentation.

### CODE REVIEW

File: schemas/npi_tables.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+* closing_of_public_venues
+* lockdown
+* non-essential_services_closure
+* religious_gatherings_banned
+* shelter_in_place
+* social_distancing
+* gathering_size_10_0
+* gathering_size_25_11
+* gathering_size_100_25
+* gathering_size_500_101
```
Comment (sglyon) [inline schemas/npi_tables.sql:18]: Should this be `_50_100`?

File: schemas/npi_tables.sql [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+* closing_of_public_venues
+* lockdown
+* non-essential_services_closure
+* religious_gatherings_banned
+* shelter_in_place
+* social_distancing
+* gathering_size_10_0
+* gathering_size_25_11
+* gathering_size_100_25
```
Comment (sglyon) [inline schemas/npi_tables.sql]: Should this be `_10_25`?

Also see below when you insert the variables into the table

**Review by sglyon — CHANGES_REQUESTED** [review]

File: schemas/npi_tables.sql (hunk 1 above)
Comment (cc7768) [reply schemas/npi_tables.sql:18]: There was a little bit of inconsistency in the naming for gathering sizes (see also the `25_11` and `100_25` categories)

Below is a list of all of the categories.

```python
In [7]: df["npi"].unique()                                                      
Out[7]: 
array(['closing_of_public_venues', 'gathering_size_10_0',
       'gathering_size_100_25', 'gathering_size_25_11',
       'gathering_size_500_101', 'lockdown',
       'non-essential_services_closure', 'religious_gatherings_banned',
       'school closure', 'shelter_in_place', 'social_distancing'],
      dtype=object)
```

**Comment by cc7768** [comment]
Also just to document the failure, the failing test was fixed by #54

File: schemas/npi_tables.sql (hunk 1 above)
Comment (cc7768) [reply schemas/npi_tables.sql:18]: I'm going to see whether it can be fixed at the raw data level

Comment (sglyon) [reply schemas/npi_tables.sql:18]: lmk when it is ready for re-review

**Comment by cc7768** [comment]
This is ready to be reviewed again. Keystone made the column rename on their end which I think is the better solution anyways.

**Pull request merged**